material-ui/material-ui-tests.tsx

```diff
@@ -223,7 +223,7 @@ const styles = {
     paddingTop: 16,
     marginBottom: 12,
     fontWeight: 400,
-  },
+  } as React.CSSProperties,
```

Comment:
would it not be better to declare `root: { justifyContent: 'space-around' as 'space-around' }` to prevent widening to `string`?